docs/Reference/jatos.js-Reference.md

```diff
@@ -1343,6 +1344,7 @@ JSON Patch add operation: Adds a value to an object or inserts it into an array.
    ```
 
    Afterwards the Batch Session contains `{"obj": {"foo": "bar"}}`.
+   Note that `jatos.batchSession.add("/obj/foo", "bar")` will fail if `"/obj"` does not already point to an object.
```

Comment:
Ah, now I understand what you mean with the innermost object missing.

I'd put this in an extra example:

1. Add to a nested object:

   If the Batch Session is `{"objA": {"objB": {}}}` and one calls

   ```javascript
   jatos.batchSession.add("/objA/objB", { foo: "bar" })
      .then(() => console.log("Batch Session was successfully updated"))
      .catch(() => console.log("Batch Session synchronization failed"));
   ```

   Afterwards the Batch Session contains `{"objA": {"objB": {"foo": "bar"}}}`.

   Note that `jatos.batchSession.add("/objA/objB", { foo: "bar" })` will fail if `"objA"` does not already point to an object.